From e8ab7aeb0b874fa5f2dfecb9826154d3ab0e35ee Mon Sep 17 00:00:00 2001
Message-Id: <e8ab7aeb0b874fa5f2dfecb9826154d3ab0e35ee.1389014116.git.minovotn@redhat.com>
In-Reply-To: <c8cc35838d42aa286242772d97e3a9be7bb786ba.1389014116.git.minovotn@redhat.com>
References: <c8cc35838d42aa286242772d97e3a9be7bb786ba.1389014116.git.minovotn@redhat.com>
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Mon, 9 Dec 2013 14:09:34 +0100
Subject: [PATCH 46/50] block: add opt_transfer_length to BlockLimits

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1386598178-11845-49-git-send-email-pbonzini@redhat.com>
Patchwork-id: 56085
O-Subject: [RHEL 7.0 qemu-kvm PATCH 48/52] block: add opt_transfer_length to BlockLimits
Bugzilla: 1039557
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>

From: Peter Lieven <pl@kamp.de>

Reviewed-by: Paolo Bonzini <pbonzini@redhat.com>
Signed-off-by: Peter Lieven <pl@kamp.de>
Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
(cherry picked from commit 7337acaf219fc988e7ba91780b11770ff95169b4)
---
 include/block/block_int.h | 3 +++
 1 file changed, 3 insertions(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 include/block/block_int.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/block/block_int.h b/include/block/block_int.h
index ab98c07..71c8b53 100644
--- a/include/block/block_int.h
+++ b/include/block/block_int.h
@@ -231,6 +231,9 @@ typedef struct BlockLimits {
 
     /* optimal alignment for write zeroes requests in sectors */
     int64_t write_zeroes_alignment;
+
+    /* optimal transfer length in sectors */
+    int opt_transfer_length;
 } BlockLimits;
 
 /*
-- 
1.7.11.7

